terraform/modules/front_door/cloudfront_access_log_bucket.tf

```diff
@@ -0,0 +1,90 @@
+#tfsec:ignore:aws-s3-enable-bucket-encryption: access log buckets not compatible with kms encryption, see https://docs.aws.amazon.com/AmazonS3/latest/userguide/troubleshooting-server-access-logging.html
+#tfsec:ignore:aws-s3-enable-bucket-logging: access log bucket doesn't need access logging itself
+#tfsec:ignore:aws-s3-encryption-customer-key: access log buckets not compatible with kms encryption, see https://docs.aws.amazon.com/AmazonS3/latest/userguide/troubleshooting-server-access-logging.html
+resource "aws_s3_bucket" "cloudfront_access_logs" {
+  #checkov:skip=CKV2_AWS_62: no need for event notifications
+  #checkov:skip=CKV_AWS_18: access log bucket doesn't need access logging itself
+  #checkov:skip=CKV_AWS_145: access log buckets not compatible with kms encryption, see https://docs.aws.amazon.com/AmazonS3/latest/userguide/troubleshooting-server-access-logging.html
+  #checkov:skip=CKV_AWS_144: cross region replication not required for access logs
+  bucket = "${var.prefix}-cloudfront-logs"
+}
+
+resource "aws_s3_bucket_public_access_block" "cloudfront_access_logs" {
+  bucket = aws_s3_bucket.cloudfront_access_logs.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+# The cloudfront access logs bucket must not use "Bucket owner enforced" as this disables bucket ACLs. Cloudfront needs to update the bucket ACLs in order to be able to deliver access logs
+# See link for info: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html#access-logs-choosing-s3-bucket
+resource "aws_s3_bucket_ownership_controls" "cloudfront_access_logs" {
+  bucket = aws_s3_bucket.cloudfront_access_logs.id
+
+  rule {
+    object_ownership = "BucketOwnerPreferred"
+  }
+}
+
+resource "aws_s3_bucket_policy" "force_ssl_and_allow_cloudfront_access_logs" {

```

Comment:
For the load balancer we had a `AllowLoadBalancerAccessLogs` statement which referenced `data.aws_elb_service_account.current.arn`. We don't have an equivalent statement here - as such should the name of this policy really include `_and_allow_cloudfront_access_logs`?

Reply:
Actually in my comment above I removed this bit of the bucket policy (allowing logs from a particular AWS service to be put in the bucket) because here we use bucket ACLs. So I should change this to something like `force_ssl_cloudfront_access_logs`. I'll update, nice find 👍

Reply:
Updated to just `force_ssl` 👍